compiler-cli/src/dependencies.rs

```diff
@@ -790,10 +803,16 @@ fn resolve_versions<Telem: Telemetry>(
     config: &PackageConfig,
     manifest: Option<&Manifest>,
     telemetry: &Telem,
+    packages_to_update: Option<&Vec<EcoString>>,
```

Comment:
I think we can take ownership of the vector as we make a copy to consume it, and it's not used after outside this function